aict_tools/io.py

```diff
@@ -522,12 +522,12 @@ def read_cta_dl1(path, aict_config, key=None, columns=None, first=None, last=Non
                         tel_table[time_key].mjd,
                         tel_pointings[time_key].mjd,
                         tel_pointings["azimuth"].quantity.to_value(u.deg),
-                    )
+                    ) * u.deg
```

Comment:
Is this really needed if we throw away the unit anyway?

Reply:
I guess its more general in case we dont want to convert to deg later

Reply:
It is needed since we need to convert to rad in `convert_units`. If the unit is not set here, `convert_units` does not convert to rad and everything goes kaputt.

Reply:
We dont convert to rad in `convert_units`, but we might so thats still a point

Reply:
Ah rigfht, sorry, just `delta_column`  is converted to rad. Anyway, better to keep the unit here with the tables.